optimum/habana/trl/trainer/ppo_trainer.py

```diff
@@ -539,11 +539,11 @@ def step(
                 active_full_logprobs = logprobs_from_logits(logits_or_none, None, gather=False)
                 ref_full_logprobs = logprobs_from_logits(ref_logits_or_none, None, gather=False)
 
-                rewards, non_score_reward = self.compute_rewards(
+                rewards, non_score_reward, kls = self.compute_rewards(
                     scores, active_full_logprobs, ref_full_logprobs, masks
                 )
             else:
-                rewards, non_score_reward = self.compute_rewards(scores, all_logprobs, ref_logprobs, masks)
+                rewards, non_score_reward, kls = self.compute_rewards(scores, all_logprobs, ref_logprobs, masks)
```

Comment:
It seems unrelated to the current PR, do you want to proceed forward this way? Or do we need another PR for this especially?

Reply:
This PR we are trying to fix all the stuff in TRL folder, so there are 2 issues: nan and then this output of compute_rewards.

changed the description to reflect this better. I think we can keep both of these in 1 PR